src/test/java/productDependency/CreateDependentCategoryTest.java

```diff
@@ -0,0 +1,193 @@
+    	
+    	dependentProductPage.clickProductsTab();
+    	dependentProductPage.clickAddCategoryButton();
+    	dependentProductPage.enterCategoryName();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	//Billing Category Product 1
+    	
+    	dependentProductPage.selectCategoryName();
+    	dependentProductPage.clickAddProductButton();
+    	dependentProductPage.enterEnglishDescription();
+    	dependentProductPage.enterProductCode();
+    	dependentProductPage.selectCompany();
+    	dependentProductPage.selectPriceCurrency();
+    	dependentProductPage.enterPriceRate();
+    	dependentProductPage.clickAddPriceButton();
+    	dependentProductPage.clickDependenciesPlus();
+    	dependentProductPage.selectProductCategoryDependency();
+    	dependentProductPage.selectProductPeriod();
+    	dependentProductPage.enterMin();
+    	dependentProductPage.clickDependencyPlusIcon();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	
+    	//Billing Category Product 2
+    	
+    	dependentProductPage.clickProductsTab();
+    	dependentProductPage.selectCategoryName();
+    	dependentProductPage.clickAddProductButton();
+    	dependentProductPage.enterEnglishDescription1();
+    	dependentProductPage.enterProductCode1();
+    	dependentProductPage.selectCompany();
+    	dependentProductPage.selectPriceCurrency();
+    	dependentProductPage.enterPriceRate1();
+    	dependentProductPage.clickAddPriceButton();
+    	dependentProductPage.clickDependenciesPlus();
+    	dependentProductPage.selectProductCategoryDependency();
+    	dependentProductPage.selectProductPeriod1();
+    	dependentProductPage.enterMin1();
+    	dependentProductPage.enterMax1();
+    	dependentProductPage.clickDependencyPlusIcon();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	
+    	//Billing Category Product 3
+    	
+    	dependentProductPage.clickProductsTab();
+    	dependentProductPage.selectCategoryName();
+    	dependentProductPage.clickAddProductButton();
+    	dependentProductPage.enterEnglishDescription2();
+    	dependentProductPage.enterProductCode2();
+    	dependentProductPage.selectCompany();
+    	dependentProductPage.selectPriceCurrency();
+    	dependentProductPage.enterPriceRate2();
+    	dependentProductPage.clickAddPriceButton();
+    	dependentProductPage.clickDependenciesPlus();
+    	dependentProductPage.selectProductCategoryDependency();
+    	dependentProductPage.selectProductPeriod2();
+    	dependentProductPage.enterMin2();
+    	dependentProductPage.enterMax2();
+    	dependentProductPage.clickDependencyPlusIcon();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	
+    	
+       	//Billing Category Product 4
+    	
+    	dependentProductPage.clickProductsTab();
```

Comment:
@mayankchhajer format the code properly all the method call and comment are aligned in a same way

Reply:
Done